components/homme/src/share/cxx/ExecSpaceDefs.cpp

```diff
@@ -49,6 +53,22 @@ void initialize_kokkos () {
   str.back() = 0;
   args.push_back(const_cast<char*>(str.data()));
 #endif
+#ifdef HIP_BUILD
```

Comment:
Noting that there are many cases of duplication, including this one, that would be avoided by unifying the preprocessor symbol and an execution space alias.